README.md

```diff
@@ -15,15 +15,22 @@ Help would be appreciated! Please join us in [slack #flaredown](https://rubyforg
 * Redis 6.2.3
 * Ruby 2.7.4 (see [RVM](https://rvm.io/) also)
 * Node 12.22.6
+## Installation
 
-You can spin up instances of the required data-stores in Docker containers using `docker compose up` in the project root.
-
-On macOS, you can install libpq by running `brew install libpq && brew link --force libpq && bundle config --local build.pg "--with-ldflags=-L$(brew --prefix libpq)/lib --with-pg-include=$(brew --prefix libpq)/include"`, which is required for `bundle install` to succeed.
+_If you are running on an M1 mac, run the following command before you start the installation process:_
+```bash
+$env /usr/bin/arch -arm64 /bin/zsh ---login
+```
```

Comment:
I don't fully understand this command :), so want to ask/confirm whether it's changing things globally or just for this shell?

FWIW i only needed to override `arch` when installing `nodejs`. Everything else was ok w/o the override.

Reply:
So what @johnpaulashenfelter discovered was that if I had preexisting gems installed on the x86 arch in my ruby folder, the set up process will continue to fail.  This step just clears out the entire gem folder to prep you to install the gems in the correct architecture.  Of course, if you have an intel mac, for example, this may not matter as much, but I mostly added it to save time on debugging.